.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: required
 rust:
 - nightly
 - beta
-- 1.0.0
+- stable
```

Comment:
I don't really have an opinion on that (I'm normally running nightly). Theoretically (I believe) 1.2 should suffice as the incompatibilty was caused by the usage of [debug builders](https://github.com/rust-lang/rust/pull/24847) which are stabilized since 1.2.